app/Branch.php

```diff
@@ -33,6 +34,56 @@ protected static function boot()
                 $branch->address = implode(", ", [$branch->address_line_1, $branch->city, $branch->postcode ." " . $branch->state]);
             }
         });
+
+        static::created(function ($branch) {
+            $access_token = session()->get('access_token');
+
+            $url = env('PARCELHUB_CENTER_URL');
+            $client = new Client();
+
+            try{
+                $response = $client->request('POST', $url.'/api/branch/create/activityHistory', [
```

Comment:
create a method in Branch model and try by passing isCreated boolean to this method